### ISSUE TITLE
[Enhancement] Update multiple licence year ends in one run. 

### ISSUE THREAD

**Issue by mdeweerd** [Lucas-C/pre-commit-hooks#94]
I have two lines with the licence year in my headers:

```text
/* Copyright (c) 2012-2023 Dummy             */
/*********************************************/
/* @copyright Copyright (c) 2012-2023 Dummy  */
```

On the first pre-commit execution, the first line is updated.
On the second pre-commit execution, the second line is updated.
On the third run, the commit is performed.

If one of the pre-commit runs could be avoided by changing both lines in a single run that would be great.

This is my configuration:

```yaml
  - repo: https://github.com/Lucas-C/pre-commit-hooks
    rev: v1.5.1
    hooks:
      - id: insert-license
        files: ^(.*\.(php))$
        args:
          - --license-filepath
          - doc/LICENSE
          - --use-current-year
          - --comment-style
          - ''
          - --no-space-in-comment-prefix
          - --fuzzy-match-extra-lines-to-check
          - '10'
          - --insert-license-after-regex
          - ^<\?php$
```

Note: My licence file is already updated like this:
```text
/* Copyright (c) 2012-2024 Dummy             */
/*********************************************/
/* @copyright Copyright (c) 2012-2024 Dummy  */
```

**Comment by mdeweerd** [comment]
Closing this as suggested in the PR - I do not have time to fix this.
I have another workaround.

**Comment by Lucas-C** [comment]
Sorry for not answering this earlier 😢

The issue is totally valid, thank you for reporting it @mdeweerd!
I'm going to reopen it.

Sorry if I was not clear in PR #94 about this:
I think it would be good to fix this,
I just don't know when I will personnally be able to look at it.

**Comment by mdeweerd** [comment]
I do not like dangling issues - nowadays bots are even closing them for us...
As you do not have the time to fix it, and me neither (because of the effort to understand the code in detail), this is one of those issues that is going to stay open forever.

The workaround is to have the same hook (and configuration) two times and use an alias for the second one.

**Comment by Lucas-C** [comment]
Implemented in PR #98
